src/libraries/System.Text.Json/src/System/Text/Json/JsonHelpers.Escaping.cs

```diff
@@ -38,7 +38,7 @@ public static byte[] GetEscapedPropertyNameSection(ReadOnlySpan<byte> utf8Value,
             int length = JsonWriterHelper.GetMaxEscapedLength(utf8Value.Length, firstEscapeIndexVal);
 
             Span<byte> escapedValue = length <= JsonConstants.StackallocThreshold ?
-                stackalloc byte[length] :
+                stackalloc byte[JsonConstants.StackallocThreshold] :
```

Comment:
Was there a request to change from `length` to `StackallocThreshold`? It seems `length` would be more correct unless I'm missing some perf (256 faster than specific number?) or other reason.

Reply:
Just part of the general "stackallocing a const value is optimized over using a variable value" work.

Reply:
The original discussion: https://github.com/dotnet/runtime/pull/54186#discussion_r661724474